sdk/cosmos/azure-cosmos/azure/cosmos/_change_feed/change_feed_state.py

```diff
@@ -208,6 +222,14 @@ def __init__(
         else:
             self._continuation = continuation
 
+        if change_feed_mode is None or change_feed_mode == ChangeFeedMode.LATEST_VERSION:
+            self._change_feed_mode = http_constants.HttpHeaders.IncrementalFeedHeaderValue
```

Comment:
hmm, why changing the changeFeedMode to be the header value. Since this value will also be used in the serialization, so I think we should stick to the changeFeedMode (enum)